### ISSUE TITLE
Header navbar dropdown not right-aligned to parent

### ISSUE THREAD

**Issue by jtreminio** [tabler/tabler#8]
Even though you're using `dropdown-menu-right` it's not right-aligning:

![image](https://user-images.githubusercontent.com/589398/37425128-52e2923c-2790-11e8-866a-e6e4a6fe6e9a.png)

It's more apparent when you remove the user's name and make some inner text longer:

![image](https://user-images.githubusercontent.com/589398/37425747-70ff3c3c-2792-11e8-8c4a-be28ab5f1fb9.png)

The reason is BS4 has `.dropdown-menu-right` nested and only works within the `.navbar-nav` class:

```
.navbar-expand {
  @each $breakpoint in map-keys($grid-breakpoints) {
    &#{$infix} {
      @include media-breakpoint-up($next) {
          .dropdown-menu-right {
            right: 0;
            left: auto; // Reset the default from `.dropdown-menu`
          }
```

However, adding it in breaks the flow:

![image](https://user-images.githubusercontent.com/589398/37425806-a9e96072-2792-11e8-83d9-34a544076c64.png)

Solution could be something like adding:

```
.nav {
  .dropdown-menu-right {
    right: 0;
    left: auto;
  }
}
```

![image](https://user-images.githubusercontent.com/589398/37425905-0a6410d2-2793-11e8-86be-6d9c1010975c.png)

But that does not fix the inner text going beyond dropdown width:

![image](https://user-images.githubusercontent.com/589398/37425926-1d859474-2793-11e8-9350-2e95dd8e1775.png)

Solution to that would be removing this line:

https://github.com/tabler/tabler/blob/master/src/assets/scss/dashboard/_dropdown.scss#L8

![image](https://user-images.githubusercontent.com/589398/37426148-93a85740-2793-11e8-97d2-2c538c3dba6d.png)

or adding:

```
.nav {
    .dropdown-menu-right {
        right: 0;
        left: auto;
    }

    .dropdown-item {
        white-space: normal;
    }
}
```

![image](https://user-images.githubusercontent.com/589398/37426509-b49f3e0e-2794-11e8-9199-7248b2c48683.png)

**Comment by codecalm** [comment]
I've fixed it in commit 0ace1b8b1abc1a6b13c20d22b1de18dbce4b5ab1

**Comment by codecalm** [comment]
fixed, online :)
